.kitchen.dokken.yml

```diff
@@ -77,45 +77,14 @@ platforms:
       - RUN zypper --non-interactive install aaa_base perl-Getopt-Long-Descriptive which net-tools
 
 suites:
-- name: server
+- name: default
   run_list:
-    - recipe[openldap-test::default]
-    - recipe[openldap::server]
-  attributes:
-    openldap:
-      rootpw: "{SSHA}6BjlvtSbVCL88li8IorkqMSofkLio58/" # secretsauce
-      basedn: "dc=example, dc=com"
-      server: "ldap.example.com"
-      preseed_dir: "/tmp/does_not_exist"
-      ssl_cert_source_cookbook: "openldap-test"
-      ssl_cert_source_path: "ldap.example.com.pem"
-      ssl_key_source_cookbook: "openldap-test"
-      ssl_key_source_path: "ldap.example.com.key"
-- name: unmanaged_ssl
+  - recipe[openldap-test::default]
+  - recipe[openldap::default]
```

Comment:
The test recipes should just do the include on this. That way they look more like true wrapper cookbooks and we move a bit more logic out of TK configs